modules/calendar/add_edit_event.php

```diff
@@ -1857,6 +1893,37 @@ function SubmitForm() {
   return true;
 }
 
+$('#reasons_modal').iziModal({
+               title: "<?php echo xlt('Reason for Cancellation'); ?>",
+               subtitle: "<?php echo xlt('Choose the reason for Cancellation.if the reason is not listed then please use the others'); ?>",
```

Comment:
place a "space" between the . and if  "Cancellation. If" and capitalize if. Change "please use the others" to "please use other"